apps/etterlatte-vedtaksvurdering-kafka/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/RapidUtsender.kt

```diff
@@ -30,6 +30,6 @@ private fun RapidInfo.sendUt(
     packet[TEKNISK_TID_KEY] = tekniskTid
     packet["vedtak"] = vedtak
     extraParams.forEach { (k, v) -> packet[k] = v }
-    context.publish(behandlingId.toString(), packet.toJson())
+    context.publish("${packet.id}:$behandlingId", packet.toJson())
```

Comment:
Usikker på om det har noe å si, men endres da key'en for meldingene, og dermed også potensiell gruppering innenfor partisjoner?

Reply:
OK, så lenge den første der er statisk så vil det vel kanskje ikke ha noe å si? 🤔